Add a Rake task to update the floor numbers for WCB

issyl0 opened the pull request:
- In AH, GDS occupied the 3rd and 6th floors. Now, in WCB,
  we occupy the 6th and 7th floors.
- This changes 6th floor to 7th floor _before_ 3rd floor to 6th floor,
  otherwise we'd change _all_ of the floors to be 7th floor.
- As all books are on shelves referenced by their `shelf_id`, we don't
  have to do a mass data migration to remap 3rd books to 6th books etc.
  Existing books now show up (tested in dev) as on 6th or 7th floor.

issyl0 commented:
Thanks @lfdebrux, I’ll merge, deploy and run this in production on Monday.

klssmith commented:
Looks good. I think we need to make sure that the task is run on Travis too - after running the task on my local test database I got a couple of test failures.

issyl0 commented:
Good catch! Will fix.

```
╭─isabelllong@gds8360 ~/repos/anthology ‹update_floor_numbers_for_wcb*›
╰─ $ RAILS_ENV=test bundle exec rake update_shelf_numbers_for_wcb
WARNING: Nokogiri was built against LibXML version 2.9.9, but has dynamically loaded 2.9.4
Updating 2 shelves...
Renamed 6th Floor to 7th Floor...
Renamed 3rd Floor to 6th Floor...
╭─isabelllong@gds8360 ~/repos/anthology ‹update_floor_numbers_for_wcb*›
╰─ $ bundle exec rake
WARNING: Nokogiri was built against LibXML version 2.9.9, but has dynamically loaded 2.9.4
Run options: --seed 53178

# Running:

.............................................E

Error:
CopyActionsTest::as a signed in user::given an available copy exists::given a shelf exists#test_0001_allow shelf to be set:
Capybara::ElementNotFound: Unable to find option "Third floor"
    test/integration/copy_actions_test.rb:54:in `block (4 levels) in <class:CopyActionsTest>'


bin/rails test test/integration/copy_actions_test.rb:46

.....E

Error:
CopyActionsTest::as a signed in user::given a copy is on loan to the signed in user#test_0002_allow the book to be returned:
Capybara::ElementNotFound: Unable to find option "Sixth floor"
    test/integration/copy_actions_test.rb:82:in `block (3 levels) in <class:CopyActionsTest>'
bin/rails test test/integration/copy_actions_test.rb:80

.......................

Finished in 5.234464s, 14.3281 runs/s, 28.4652 assertions/s.
75 runs, 149 assertions, 0 failures, 2 errors, 0 skips
```

issyl0 commented:
@klssmith This is good to go now, I think. Shall I squash the commits?

Also I'll need your help for running it on PaaS, I have no idea.

klssmith commented:
@issyl0 I think we can just deploy as normal, then ssh onto the app and run the rake task from there. Looks like you don't have access to the right PaaS space - I'm happy to do it or we can do it together
